portal/src/it/scala/cromwell/pipeline/controller/ProjectControllerItTest.scala

```diff
@@ -58,12 +58,12 @@ class ProjectControllerItTest
         }
       }
 
-      "return status code 403 if user isn`t project owner" in {
+      "return a project with the same name and a random userId" in {
```

Comment:
What do we recive if project will only for stranger user ?

Reply:
Thanks for this question! We should get either the project if there is a correct project name, or nothing (status code 404) if we transfer wrong name.